pyiceberg/io/pyarrow.py

```diff
@@ -761,6 +761,32 @@ def primitive(self, primitive: pa.DataType) -> T:
         """Visit a primitive type."""
 
 
+class PreOrderPyArrowSchemaVisitor(Generic[T], ABC):
```

Comment:
```suggestion
class _PreOrderPyArrowSchemaVisitor(Generic[T], ABC):
```

Reply:
Good catch - I missed this one. I'll remove this Visitor as its no longer in use